src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -53,6 +58,9 @@ function MoneyRequestSelectorPage(props) {
         [CONST.IOU.MONEY_REQUEST_TYPE.SEND]: translate('iou.sendMoney'),
         [CONST.IOU.MONEY_REQUEST_TYPE.SPLIT]: translate('iou.splitBill'),
     };
+    const isFromGlobalCreate = !props.route.params.reportID;
+    const isExpenseRequest = props.report.chatType === CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT;
+    const shouldDisplayDistanceRequest = (canUseDistanceRequests && isExpenseRequest) || isFromGlobalCreate;
```

Comment:
Request from FAB menu doesn't depend on beta permission?
```suggestion
    const shouldDisplayDistanceRequest = canUseDistanceRequests && (isExpenseRequest || isFromGlobalCreate);
```

Reply:
good catch!!! you saved my life!